### PR TITLE
Adding basic style for view details page

### LINKED ISSUE #6: Add basic styling for ViewServiceDetails page

### PR DESCRIPTION
**Pull request by cijokb**
Added inline source mapping in webpack 
Added basic styling and card 
fixes #6

### CODE REVIEW

File: src/components/ServiceDetails/ServiceDetails.css [hunk 1]
```diff
@@ -17,4 +17,14 @@
...
     }
+
+    table {
+        color: #1f2939;
+        font-size: 14px;
+        width: 100%;
+    }
+    td {
+        width: 70%;
+        padding: 0 0 9pt;
```
Comment (ivinjose) [inline src/components/ServiceDetails/ServiceDetails.css]: Try to use pixels across our project.

File: src/components/ServiceDetails/ServiceDetails.js [hunk 2]
```diff
@@ -3,34 +3,57 @@ import ReactDOM from 'react-dom';
...
 import globalStyles from '../../styles/global.css';
+import {Card, CardHeader, CardText} from 'material-ui/Card';
 
 class ServiceDetails extends React.Component {
 	constructor() {
 		super();
 	}
- 
+
+  formatDate(date){
```
Comment (ivinjose) [inline src/components/ServiceDetails/ServiceDetails.js]: Remove if not needed.

File: src/components/ServiceDetails/ServiceDetails.js [hunk 3]
```diff
@@ -3,34 +3,57 @@ import ReactDOM from 'react-dom';
...
+	   return new Date(date).toLocaleDateString(); // Todo
+    };
 	render() {
-		if( this.props.data.length == 0 ){
+        const me =this;
```
Comment (ivinjose) [inline src/components/ServiceDetails/ServiceDetails.js]: Is this properly indexed? Try to follow tabular indexing across the code.

Comment (cijokb) [reply src/components/ServiceDetails/ServiceDetails.js]: yes. It seems git issue

Comment (ivinjose) [reply src/components/ServiceDetails/ServiceDetails.js]: Ok.

**Pull request merged**